### ISSUE TITLE
[Feature request] Support Nvidia RTX Remix

### ISSUE THREAD

**Issue by mirshekarm** [RPCS3/rpcs3#14806]
### Quick summary

Hello, I wanted to see if it is possible to support Nvidia RTX Remix for older games?  If it is possible, it will be possible to give more attractive graphics to a masterpiece game like God of War and it will be great!

**Comment by AniLeo** [comment]
No, this is an emulator platform running on Vulkan, not one single game nor native D3D8 or D3D9
